src/main/java/ca/bc/gov/chefs/etl/forms/pcd/haHierarchy/model/Community.java

```diff
@@ -12,6 +12,7 @@ public class Community implements IModel {
     private String communityName;
     private String hsiarServicePlanGapAnalysis;
     private String hsiarServicePlanGapAnalysisDate;
+    private String pcnCommunityCode;
```

Comment:
I'm following up to see if it's too late to change the column name but I prefer communityCode.

Reply:
This rename should be resolved in the latest commit!